releasenotes/notes/su2-synthesis-295ebf03d9033263.yaml

```
@@ -0,0 +1,7 @@
+---
+features:
+  - |
+    The new function `mcsu2_real_diagonal(circuit, unitary, controls, target, ctrl_state)` 
+    applies a multi-controlled SU(2) gate unitary with one real diagonal. With n-1 controls, 
+    the decomposition requires approximately 16n cx gates. Synthesis of mcrx and mcry without 
+    auxiliary qubits now uses the `mcsu2_real_diagonal` decomposition.
```

Since this function is not in the API documentation, I think we don't have to mention it in the releasenotes. Instead we could rephrase this as bugfix. Also, we should include the deprecation. Here's an example, but feel free to rephrase as is best!
```suggestion
fixes:
  - |
    Fixed the gate decomposition of multi-controlled Z rotation gates added via
    :meth:`.QuantumCircuit.mcrz`. Previously, this method implemented a multi-controlled
    phase gate, which has a relative phase difference  to the Z rotation.
deprecation:
  - |
    Deprecated the `use_basis_gates` argument in the multi-controlled Pauli rotation methods,
    :meth:`.QuantumCircuit.mcrx`, :meth:`.QuantumCircuit.mcry` and :meth:`.QuantumCircuit.mcrz`.
    To convert to a certain basis, you can instead use the :func:`.transpile` function with the 
    ``basis_gates`` argument, or the :class:`.BasisTranslator` transpiler pass.
```

---

This release note appears in #9688, I have written a new release note for this PR.